.github/workflows/deploy.yml

```
@@ -18,26 +18,25 @@ jobs:
-      - name: Checkout
-        uses: actions/checkout@v6
+      - uses: actions/checkout@v6
+        id: checkout
         with:
           persist-credentials: false
 
-      - name: Setup Bun
-        uses: ./.github/actions/setup-bun
+      - uses: ./.github/actions/setup-bun
+        id: setup
         with: { install-node: "true" }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)